app/views/glossary_terms/show.html.erb

```diff
@@ -2,11 +2,13 @@
   @title = :show_glossary_term_title.t(name: @glossary_term.name)
 
   tabs = [
-    link_to(:glossary_term_index.t, action: :index),
-    link_to(:create_glossary_term.t, action: :create_glossary_term),
-    link_to(:edit_glossary_term.t, action: :edit_glossary_term,
-            id: @glossary_term.id)
+    link_to(:glossary_term_index.t, glossary_terms_path),
+    link_to(:create_glossary_term.t, new_glossary_term_path),
+    link_to(:edit_glossary_term.t, edit_glossary_term_path(@glossary_term.id))
   ]
+  tabs << destroy_button(@glossary_term) if in_admin_mode?
```

Comment:
Button styling is a smidgen off. (I want it to look exactly like a link.)
I can't figure out how to fix it.

Reply:
What's off?  It looks very similar to the links to me.  The color is a bit different, but that's easy to fix in the SCSS file.

Reply:
I will change only the hover, and leave it at that.
Weight is correct, and I think the color is correct. (@mo-nathan is right about those.)